src/main/java/com/github/maxopoly/MemeMana/command/CmdMana.java

```diff
@@ -0,0 +1,91 @@
+package com.github.maxopoly.MemeMana.command;
+
+import vg.civcraft.mc.civmodcore.command.PlayerCommand;
+import vg.civcraft.mc.namelayer.NameAPI;
+import com.devotedmc.ExilePearl.ExilePearlPlugin;
+import com.devotedmc.ExilePearl.ExilePearl;
+import com.github.maxopoly.MemeMana.model.MemeManaPouch;
+import com.github.maxopoly.MemeMana.model.MemeManaOwner;
+import com.github.maxopoly.MemeMana.MemeManaIdentity;
+import com.github.maxopoly.MemeMana.MemeManaPlugin;
+import org.bukkit.entity.Player;
+import org.bukkit.command.CommandSender;
+import java.util.UUID;
+import java.util.List;
+import java.util.LinkedList;
+
+public class CmdMana extends PlayerCommand {
+	public CmdMana(String name) {
+		super(name);
+		setIdentifier("mana");
+		setDescription("The base command for Meme Mana");
+		setUsage("\n/mana show -- Show your current mana\n/mana refill [Amount] -- Refill the pearl in your hand by an amount\n/mana transfer <Player> <Amount> -- Transfer some mana to another player");
```

Comment:
Fine for now, but I'd prefer breaking this up into shorter commands with one functionality each. Makes it easier to use, allows easy individual tab completion and keeps the usage from being a book. 

Not relevant to this, but another feature for the future, that I want to add is tying a mana pouch to chests, so it'll automatically feed pearls inside of those chests, assuming sufficient mana exists. Those players already login and play enough to generate the mana, so why make them dig out the pearls and run clunky commands?

Reply:
Pretty sure the reason why crafting to refill exists is to prevent this kind of automation, so definitely check with Teal.

Also, I was talking with Teal, and we wanted to move ExilePearl's command API to CivModCore, at which point I'll split this up properly, since that API handles this very nicely. It actually looks like it was designed to be a drop in replacement for CivModCore's thing.